client/src/components/MainPage/MainContents/Log/index.tsx

```diff
@@ -0,0 +1,269 @@
+    done: true,
+    labels: [
+      {
+        idx: 0,
+        color: 'red',
+        title: '고통',
+        amount: 10,
+        unit: 'Kg',
+      },
+    ],
+  },
+  {
+    idx: 1,
+    title: '육아',
+    importance: 5,
+    startedAt: '00:00',
+    endedAt: '23:59',
+    lat: 123,
+    lng: 123,
+    location: '집',
+    public: true,
+    tag_idx: 1,
+    tag_name: '육아',
+    content: '아 힘들다',
+    done: false,
+    labels: [
+      {
+        idx: 1,
+        color: 'yellow',
+        title: '귀여움',
+        amount: 20,
+        unit: 'Kg',
+      },
+    ],
+  },
+  {
+    idx: 2,
+    title: '페어저쩌구',
+    importance: 3,
+    startedAt: '13:00',
+    endedAt: '16:30',
+    lat: 123,
+    lng: 123,
+    location: '줌',
+    public: true,
+    tag_idx: 0,
+    tag_name: '부스트캠프',
+    content: '아 재미있다',
+    done: false,
+    labels: [
+      {
+        idx: 3,
+        color: 'blue',
+        title: '공부',
+        amount: 120,
+        unit: '분',
+      },
+    ],
+  },
+  {
+    idx: 3,
+    title: '분유먹이기',
+    importance: 4,
+    startedAt: '12:00',
+    endedAt: '12:30',
+    lat: 123,
+    lng: 123,
+    location: '집',
+    public: true,
+    tag_idx: 1,
+    tag_name: '육아',
+    content: '맛있을까?',
+    done: true,
+    labels: [
+      {
+        idx: 1,
+        color: 'yellow',
+        title: '귀여움',
+        amount: 20,
+        unit: 'Kg',
+      },
+      {
+        idx: 2,
+        color: 'green',
+        title: '분유',
+        amount: 200,
+        unit: 'ml',
+      },
+    ],
+  },
+  {
+    idx: 4,
+    title: '넷플릭스보기',
+    importance: 2,
+    startedAt: '20:00',
+    endedAt: '21:30',
+    lat: 123,
+    lng: 123,
+    location: '집',
+    public: true,
+    tag_idx: 2,
+    tag_name: '일상',
+    content: '요즘은 뭐가 재미있지?',
+    done: false,
+    labels: [
+      {
+        idx: 4,
+        color: 'pink',
+        title: '휴식',
+        amount: 900,
+        unit: '분',
+      },
+    ],
+  },
+  {
+    idx: 5,
+    title: '다섯번째',
+    importance: 2,
+    startedAt: '20:00',
+    endedAt: '21:30',
+    lat: 123,
+    lng: 123,
+    location: '집',
+    public: true,
+    tag_idx: 3,
+    tag_name: '생존',
+    content: '요즘은 뭐가 재미있지?',
+    done: false,
+    labels: [
+      {
+        idx: 4,
+        color: 'pink',
+        title: '휴식',
+        amount: 900,
+        unit: '분',
+      },
+    ],
+  },
+  {
+    idx: 6,
+    title: '여섯번째',
+    importance: 2,
+    startedAt: '20:00',
+    endedAt: '21:30',
+    lat: 123,
+    lng: 123,
+    location: '집',
+    public: true,
+    tag_idx: 0,
+    tag_name: '부스트캠프',
+    content: '요즘은 뭐가 재미있지?',
+    done: false,
+    labels: [
+      {
+        idx: 4,
+        color: 'pink',
+        title: '휴식',
+        amount: 900,
+        unit: '분',
+      },
+    ],
+  },
+];
+const Log = () => {
+  const [selectedElement, setSelectedElement] = useState<number | null>(null);
+  const selectedRef = useRef<HTMLDivElement | null>(null);
+  const [mousePos, setMousePos] = useState<number[]>([0, 0]);
```

Comment:
마우스 좌표는 배열보다 객체로 관리하는 편이 이해하기 좋을 것 같습니다. `mousePos[0]` → `mousePos.x`
```suggestion
  const [mousePos, setMousePos] = useState<Position>({x: 0, y: 0});
```